packages/plugin-lighthouse/src/lib/lighthouse-plugin.unit.test.ts

```diff
@@ -17,47 +17,36 @@ describe('lighthousePlugin-config-object', () => {
     ]);
   });
 
-  it('should filter audits by onlyAudits string "first-contentful-paint"', () => {
+  it('should mark audits in onlyAudits as not skipped', () => {
     const pluginConfig = lighthousePlugin('https://code-pushup-portal.com', {
       onlyAudits: ['first-contentful-paint'],
     });
 
     expect(() => pluginConfigSchema.parse(pluginConfig)).not.toThrow();
-
-    expect(pluginConfig.audits[0]).toEqual(
+    expect(
+      pluginConfig.audits.find(({ slug }) => slug === 'first-contentful-paint'),
+    ).toEqual(
       expect.objectContaining({
-        slug: 'first-contentful-paint',
+        isSkipped: false,
       }),
     );
   });
```

Comment:
I'm missing some assertion that the other audits have `isSkipped: true`. (Same for groups.)